testing/test_deploy.py

```diff
@@ -446,7 +446,7 @@ def deploy_minikube(args):
       raise
     content = json.loads(e.content)
     if content.get("error", {}).get("code") == requests.codes.CONFLICT:
-      # We don't want to keep going so we reraise the error after logging
+      # We don't want to keep going so we raise the error after logging
```

Comment:
this is intended to be reraise (or re-raise), right?

Reply:
'reraise' may be the better choice.

Reply:
'reraise' may be the better choice.